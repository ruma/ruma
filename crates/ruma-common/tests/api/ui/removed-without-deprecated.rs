use ruma_common::{api::Metadata, metadata};

const METADATA: Metadata = metadata! {
    method: GET,
    rate_limited: false,
    authentication: None,
    history: {
        unstable => "/a/path",
        1.1 => removed,
    }
};

fn main() {}
