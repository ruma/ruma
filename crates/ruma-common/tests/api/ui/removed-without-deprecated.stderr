error: no rules expected the token `history_impl`
  --> tests/api/ui/removed-without-deprecated.rs:3:28
   |
3  |   const METADATA: Metadata = metadata! {
   |  ____________________________^
4  | |     method: GET,
5  | |     rate_limited: false,
6  | |     authentication: None,
...  |
10 | |     }
11 | | };
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::metadata` which comes from the expansion of the macro `metadata` (in Nightly builds, run with -Z macro-backtrace for more info)
